Revise Swift entries (#929)
Codecademy/docs: content/swift/concepts/variables/variables.md

  - 'paths/build-ios-apps-with-swiftui'
---

> The Swift programming language allows values to be stored in two types of containers: `var` for variables and `let` for constants.

Reviewer: ```suggestion
The Swift programming language allows values to be stored in two types of containers: `var` for **variables** and `let` for constants.
```

Author: Also bolded constants

As the names imply, a variable's value can change or reassigned later in the program, while a constant's value cannot. It is best practice to use constants as much as possible to prevent accidentally reassigning values later on in a program.

> **Note:** Once a variable or constant's data type has been established, whether explicitly or implicitly (inferred by the compiler) declared, it cannot be changed. In addition, the variable name cannot be used for another data type within the same program. In the same manner, a variable cannot be changed into a constant and vice versa.

## Declaring and Initializing Variables and Constants

> Variables and constants do not require initilization when declared. However, the variable or constant requires type annotation so thatwhen the compiler reads the code line-by-line, it can determine the data type at the time of the build. A `Use of undeclared type` error will be thrown otherwise. Swift also allows a wide range of characters to be used as constant and variable names including Unicode characters and emojis.

Reviewer: ```suggestion
Variables and constants do not require initialization when declared. However, the variable or constant requires type annotation so that when the compiler reads the code line-by-line, it can determine the data type at the time of the build. A `Use of undeclared type` error will be thrown otherwise. Swift also allows a wide range of characters to be used as constant and variable names including Unicode characters and emojis.
```

## Accessing and Reassigning Variables and Constants

> The value of a variable can be changed by using the assignment operator to set the value to a different value of the same type. You can also use another variable to set the value of a variable.

Reviewer 2: ```suggestion
The value of a variable can be changed by using the [assignment operator](https://www.codecademy.com/resources/docs/swift/operators) to set the value to a different value of the same type. You can also use another variable to set the value of a variable.
```
